.ci/pipeline/docs.yml

```diff
@@ -16,26 +16,47 @@ pr:
     - doc
     - examples
 
+variables:
+  - name: python.version
+    value: '3.7'
+  - name: daal4py.version
+    value: '2021.2.0'
```

Comment:
I think we will have problems here and will forgot to update version here.

Reply:
agree